boa/contracts/vyper/vyper_contract.py

```diff
@@ -467,6 +476,20 @@ def __repr__(self):
         return repr(self.dump())
 
 
+# data structure to represent the constants in a contract
+class ConstantsModel:
+    def __init__(self, compiler_data: CompilerData):
+        for k, v in compiler_data.vyper_module_unfolded._metadata["namespace"].items():
+            if isinstance(v, VarInfo) and v.is_constant and v.decl_node:
+                setattr(self, k, v.decl_node.value.value)
```

Comment:
this can be wrong if the constant is complex, e.g. `FOO: constant(uint256) = BAR + BAZ`

Reply:
in 0.4.0, we would use `decl_node.reduced().value`